src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs

```diff
@@ -86,6 +90,50 @@ private Task<AreaOfInterestResult> ComputerVisionGetAreaOfInterest(string url)
             return _computerVisionClient.GetAreaOfInterestAsync(url);
         }
 
+        private async Task<TextOperationResult> ComputerVisionRecognizedText(string url, string recognizeTextMode)
+        {
+            var parsedRecognizeTextMode = GetTextRecognitionMode(recognizeTextMode, TextRecognitionMode.Printed);
+            var recognizeTextHeaders = await _computerVisionClient.RecognizeTextAsync(url, parsedRecognizeTextMode);
+
+            // Retrieve the URI where the recognized text will be stored from the Operation-Location header
+            var operationLocation = recognizeTextHeaders.OperationLocation;
+            var operationId = operationLocation.Substring(operationLocation.Length - _computerVisionOperationIdLength);
+
+            var result = await _computerVisionClient.GetTextOperationResultAsync(operationId);
+
+            // Wait for the operation to complete
+            var i = 1;
```

Comment:
Suggesting to rename to `iteration` for clarity.

Reply:
Changing it in an incoming update